### DISCUSSION TITLE
Custom media files path

### DISCUSSION THREAD

**Discussion by MarekTrso** [wintercms/winter#692] [Q&A]
Hi, I’m new to Winter CMS and I would like to create something like blog page. I’m using Rainlab Blog plugin for creating posts.

My problem is that I want to store my media (images, videos, etc. ) in specific folders for each blog post. For example if the name of post is “Example Post”, then I would like to have all media of that post stored in folder called “ExamplePost”.

I know that it’s stored in storage/app/media (when Media Finder is used) and storage/app/uploads (for File Upload), but than it creates som folders based on disk_name property and I don’t understand that.

Is it possible to change/rename that folders with the ones that I want?

Thank you for all your help.

**Comment by AIC-BV** [comment]
Its not the correct answer to your question; but you should use [Winter.Blog](https://github.com/wintercms/wn-blog-plugin) instead of Rainlab Blog, for future compatibility. It is basically the same plugin, but this one is supported by the WinterCMS team.

**Reply by MarekTrso** [reply]
Thank you for your advice :)
